zilencer/views.py

```diff
@@ -194,10 +194,10 @@ def unregister_remote_push_device(
     validate_bouncer_token_request(token, token_kind)
     user_identity = UserPushIdentityCompat(user_id=user_id, user_uuid=user_uuid)
 
-    deleted = RemotePushDeviceToken.objects.filter(
+    (num_deleted, ignored) = RemotePushDeviceToken.objects.filter(
         user_identity.filter_q(), token=token, kind=token_kind, server=server
```

Comment:
Can unpack this as `(num_deleted, _) = ...` as well, if `ignored` is not used anywhere later :)

Reply:
Done -- Thanks!

Reply:
We try hard to avoid using `_` for this variable, since Django uses that for translations, and it can produce unfortunate results to rebind that within a function.

Reply:
Oh, makes sense! Would it be worth adding a linter rule to catch the `, _) =` pattern?

Reply:
Also, for `get_or_create()` queries, the tuple returned has been handled differently. 

These are some of the patterns:
```python
object, _ = ModelName.objects.get_or_create()
object, created = ... (even if 'created' is not used later)
(object, _) = ...
(object, created) = ...
```
Should we keep things uniform as:
```python
(object, created) = ...
(object, ignored) = ...
```
or it a lot of nitpicking? :)

Reply:
Yeah, that sort of linter rule would be a good idea; this regular expression shows we have very violations right now, at least:

```
$ git grep ', _[)] = '
zerver/lib/test_classes.py:        (sending_client, _) = Client.objects.get_or_create(name=sending_client_name)
zerver/lib/test_classes.py:        (sending_client, _) = Client.objects.get_or_create(name=sending_client_name)
zerver/lib/test_classes.py:        (sending_client, _) = Client.objects.get_or_create(name=sending_client_name)
zerver/models.py:    (client, _) = Client.objects.get_or_create(name=name[0 : Client.MAX_NAME_LENGTH])
zerver/webhooks/beanstalk/view.py:        (short_commit_msg, _, _) = payload["message"].tame(check_string).partition("\n")
```

Do you know how to do this using `tools/linter_lib/custom_check.py`?

Reply:
I will do this.